### DISCUSSION TITLE
Loading Checkpoints in openllm

### DISCUSSION THREAD

**Discussion by vdeeplearn** [bentoml/OpenLLM#253] [General]
How can I load checkpoints directory containing the model files for llama2 loading

Similar to the below script can I specify the model checkpoint directory?

from langchain.llms import OpenLLM

llm = OpenLLM(
    model_name="dolly-v2",
    model_id="databricks/dolly-v2-3b",
    temperature=0.94,
    repetition_penalty=1.2,
)

Is there any info in the documentation of openllm

**Comment by aarnphm** [comment]
`model_id` can accept path
I'm working on documentation atm
